src/middlewared/middlewared/plugins/zettarepl.py

```diff
@@ -709,6 +709,14 @@ async def _replication_task_definition(self, pools, replication_task):
         if replication_task["lifetime_value"] is not None and replication_task["lifetime_unit"] is not None:
             definition["lifetime"] = lifetime_iso8601(replication_task["lifetime_value"],
                                                       replication_task["lifetime_unit"])
+        if replication_task["lifetimes"]:
+            definition["lifetimes"] = {
+                f"lifetime_{i}": {
+                    "schedule": zettarepl_schedule(lifetime["schedule"]),
+                    "lifetime": lifetime_iso8601(lifetime["lifetime_value"], lifetime["lifetime_unit"]),
```

Comment:
Will `lifetime_value` and `lifetime_unit` always exist ?

Reply:
You are right, I forgot `required` in the `Dict`, thank you!